modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/BooleanObjectFieldBusinessType.java

```diff
@@ -57,7 +58,7 @@ public PropertyDefinition.PropertyType getPropertyType() {
 
 	@Override
 	public boolean isLocalizable() {
-		return true;
+		return FeatureFlagManagerUtil.isEnabled("LPD-32050");
```

Comment:
@pedro-oliveira446 We should consider the companyId to check the feature flag. I see that we don't have this info in this level, so remove this check from here and place it just where it's being used.

Reply:
Just done!